NeuroID/Utilities/NIDConfigService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NIDConfigService.swift
+//  NeuroID
+
+import Foundation
+import Alamofire
+
+internal class NIDConfigService {
+    
+    struct ResponseData: Decodable {
+        var callInProgress: Bool = true
+        var geoLocation: Bool = true
+        var eventQueueFlushInterval: Int = 5
+        var eventQueueFlushSize: Int = 2000
+        var requestTimeout: Int = 10
+        var gyroAccelCadence: Bool = true
```

Comment:
Is the v1 implementation list at the end of the design doc the correct default values?  `gyro_accel_cadence` is marked as `false`.  Can you confirm the correct default?  @kishisaka-nid what is android default?

Reply:
I misread. Good catch